pkg/aws_config_server/webserver.go

```diff
@@ -52,23 +52,23 @@ func requireAuthentication(next httprouter.Handle, verifier oidcVerifier) httpro
 		authHeader := r.Header.Get("Authorization")
 		ctx := r.Context()
 		if len(authHeader) <= 0 {
-			logrus.Warn("error: No Authorization header found.")
+			logrus.Error("error: No Authorization header found.")
 			http.Error(w, fmt.Sprintf("%v:%s", 407, http.StatusText(407)), 407)
 			return
 		}
 		rawIDToken := stripBearerPrefixFromTokenString(authHeader)
 
 		idToken, err := verifier.Verify(ctx, rawIDToken)
 		if err != nil {
-			logrus.Warnf("error: Unable to verify idToken. %s", err)
+			logrus.Errorf("error: Unable to verify idToken. %s", err)
```

Comment:
```suggestion
			logrus.Warnf("error: Unable to verify idToken. %s", err)
```
Same here, this should'nt trigger a sentry error.